mod/wikirate/set/type_plus_right/wikirate_company/general_overview.rb

```diff
@@ -0,0 +1,4 @@
+
+view :missing do |_args|
+  link_to 'Add General Overview', { view: :new }, remote: false
+end
```

Comment:
I remember at some moment we wanted to have the add about to link to a new card page instead of creating inside the slot. 

Do we still want to keep that?